ts/src/bybit.ts

```diff
@@ -1148,6 +1148,18 @@ export default class bybit extends Exchange {
         return this.milliseconds () - this.options['timeDifference'];
     }
 
+    addPaginationCursorToResult (response) {
```

Comment:
@jorisw I noticed we were repeating the same thing repeatedly, so I added this method to DRY it.